tests/scripts/components-configuration-crypto.sh

```diff
@@ -35,9 +35,13 @@ component_test_crypto_with_static_key_slots() {
     msg "build: crypto full + MBEDTLS_PSA_STATIC_KEY_SLOTS"
     scripts/config.py crypto_full
     scripts/config.py set MBEDTLS_PSA_STATIC_KEY_SLOTS
+    # Intentionally set MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE slightly smaller
+    # than PSA_EXPORT_KEY_PAIR_OR_PUBLIC_MAX_SIZE where the latter would be 2364
+    # bytes for an RSA key pair of 4096 bits.
+    scripts/config.py set MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE 2362
```

Comment:
The size of an RSA key pair representation is slightly variable for a given bit-size, because the format requires removing leading zeros. Given that the test case tries to generate a random key, if the cutoff is only a few bytes below the maximum size, the test case will sometimes fail randomly. We can take a much lower cutoff here (as well as a lower key size, see my comment on the test case). What we want is, in that build, to have at least one test case that successfully creates an RSA key pair, and another one that returns `NOT_SUPPORTED` when doing the same thing but with a larger bit-size that is less than `PSA_VENDOR_RSA_MAX_KEY_BITS`.

Reply:
> The size of an RSA key pair representation is slightly variable for a given bit-size, because the format requires removing leading zeros

Oh, I didn't check the PSA format used to store RSA keys and I assumed it just worked the same way as EC keys (where, if I'm not wrong, the size is always the same and it's given from the bit-size).
Thanks for the notification. I will fix this